guides/source/active_record_migrations.md

```diff
@@ -436,8 +436,17 @@ change_column_default :products, :approved, false
 This sets `:name` field on products to a `NOT NULL` column and the default
 value of the `:approved` field to false.
 
-TIP: Unlike `change_column` (and `change_column_default`), `change_column_null`
-is reversible.
+Additionaly, for `change_column_default`, you can pass a Hash which defines the
```

Comment:
We should encourage writing reversible migrations by default. Let's change default documentation to use `from:` and `to:`, and here write something like:
 
```Note: You could also write the above migration as `change_column_default :products, :approved, false`, but unlike the previous example, this would make your migration irreversible.```